core/src/test/java/org/elasticsearch/search/sort/FieldSortIT.java

```diff
@@ -1339,68 +1334,6 @@ public void testSortOnRareField() throws IOException {
         assertThat(searchResponse.getHits().getAt(2).sortValues()[0], equalTo("03"));
     }
 
-    public void testSortMetaField() throws Exception {
-        XContentBuilder mapping = XContentFactory.jsonBuilder().startObject().startObject("type")
-                .startObject("_timestamp").field("enabled", true).endObject()
-                .endObject().endObject();
-        assertAcked(prepareCreate("test")
-            .addMapping("type", mapping));
-        ensureGreen();
-        final int numDocs = randomIntBetween(10, 20);
-        IndexRequestBuilder[] indexReqs = new IndexRequestBuilder[numDocs];
-        for (int i = 0; i < numDocs; ++i) {
-            indexReqs[i] = client().prepareIndex("test", "type", Integer.toString(i)).setTimestamp(Integer.toString(randomInt(1000)))
-                    .setSource();
-        }
-        indexRandom(true, indexReqs);
-
-        SortOrder order = randomFrom(SortOrder.values());
-        SearchResponse searchResponse = client().prepareSearch()
-                .setQuery(matchAllQuery())
-                .setSize(randomIntBetween(1, numDocs + 5))
-                .addSort("_uid", order)
```

Comment:
I think I'll mave it to TimestampTTLUpdateIT and rename it to something like TimestampTTLBWIT